site/source/docs/api_reference/stack.h.rst

```diff
@@ -0,0 +1,39 @@
+.. _stack-h:
+
+=======
+stack.h
+=======
+
+The functions defined in `<emscripten/stack.h>` allow inspecting
+information about the WebAssembly data stack. This data stack is
+the data contained within the linear memory.
```

Comment:
There is often confusion on this, so might be good to clarify?

```suggestion
information about the WebAssembly data stack (sometimes called the "user stack"
or the "c stack"). This data stack is the data contained within the linear
memory (as opposed to the trusted call stack that is managed by the VM, and
which is not visible to the running program).
```

Reply:
Updating and force-pushing. Your text is much better than mine here!